app-script-email-for-form-submission/README.md

```diff
@@ -0,0 +1 @@
+#lab2
```

Comment:
please update Readme file with more information like link to the form, what it does